chart/values.yaml

```diff
@@ -55,6 +55,24 @@ iq_server:
     # password
     passwordSecret:
 
+iq_server_jobs:
+  # We usually recommend not to specify default resources and to leave this as a conscious choice for the user.
+  # This also increases the chance that the chart runs on environments with little resources.
+  # The commented values below are the minimum recommended resources for production.
+  resources:
+    requests:
+      cpu: #8
+      memory: #8Gi
```

Comment:
I know it's just a comment, but I would make this less. A user specifying 8g of requested memory here following the comment could be dangerous.

Same for CPU. Maybe try 500m and 1g?

Reply:
Done.